net-irc/ngircd/files/ngircd-26.1-configure-getaddrinfo.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/configure b/configure
```

Comment:
Make sure to add an explanation here why we're only patching generated configure & bug links.